templates/repo/home.tmpl

```diff
@@ -136,7 +138,7 @@
 					</div>
 					{{template "repo/cite/cite_modal" .}}
 				{{end}}
-				{{if and (not $isHomepage) (not .IsViewFile) (not .IsBlame)}}
+				{{if and (not $isHomepage) (not .IsViewFile) (not .IsBlame)}}{{/* IsViewDirectory (not home) */}}
```

Comment:
Add a variable for IsViewDirectory?

Reply:
Don't think so, as this is a quick bug fix, do not want to introduce any potential regression.

Reply:
So we need to remember adding `(not .IsXXX)` here when adding a new page?

Reply:
I think we need to rewrite the page at that time, instead of introducing more `if` tricks.

Reply:
Agree. So can you add TODO or FIXME here, for avoiding forgetting this.

Reply:
```suggestion
				{{if and (not $isHomepage) (not .IsViewFile) (not .IsBlame)}}{{/* IsViewDirectory (not home), TOD: split the templates, avoid use "if" tricks */}}
```

Reply:
You forgot to commit the suggestion.

Reply:
Really ? I do not think I forgot. Why I have to keep explaining everything .........

![image](https://github.com/go-gitea/gitea/assets/2114189/632cca8d-5dbf-4250-b6de-0bde58b5ebde)

Reply:
Oh ok, I missed https://github.com/go-gitea/gitea/pull/30678/commits/b3829ad25c75db883e18ac64155cc06cf7ec9e86.